### DISCUSSION TITLE
Export & analyze data

### DISCUSSION THREAD

**Discussion by rbruno1** [jspsych/jsPsych#1977] [Q&A]
What are some of the best ways or software to use to export and analyze data. My experiment uses audio button responses and it is a bit tedious to go through and translate the numbers to responses. Is there a way to do all of this automatically?

**Comment by acer-helia** [comment]
Hi, is your data stored in JSON format, or CSV? If it's in JSON you can use the online converter: https://data.page/json/csv or if in CSV you can either create a function in Excel to automatically convert audio responses numbers to responses. Otherwise I use R to process JSON data - there are some tutorials on stack exchange on how to do this, and in R you can write a function to automatically convert responses as well.

**Reply by rbruno1** [reply]
it is in csv. I am trying to use R to give me a summary of each run (defined by the run_ID), but it gives me a summary of the entire dataset, which is not useful in my case. any idea how to do this?

**Reply by acer-helia** [reply]
Can you post a snippet of your csv file and how it is set up? It's helpful if you can give us a sense of what the data currently looks like.

**Reply by rbruno1** [reply]
[pod-piloting (4).csv](https://github.com/jspsych/jsPsych/files/6780015/pod-piloting.4.csv)
I am trying to automatically generate data on whether the participant found the task easy or difficult (0/1 --> Easy/difficult), the participants' age range (0/1/2/3 --> 10 or below/11-18/19-25/25 and above), english native (0/1 --> Yes/No), as well as accuracy data on each condition(each participant will try to discriminate the first syllable of bucket and buckle, garlic and garbage, marble and marker, peanut and peacock, turtle and turkey). So there are 5 conditions in each run that i want to generate this data for. So my analysis program will have to recognize each run as a distinct run, as well as each condition as a distinct condition.
